src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -225,7 +225,7 @@ namespace JSONHelpers
                 }
             }
 
-            DeleteFileW(tmpFilePath.c_str());
```

Comment:
File deletion looks like side-effect here. One function should do one thing. When you see `Parse...`, you expect that it only parses, and other things like file deletion are unexpected there.

Reply:
@bzoz Already pointed  this out. So far I've just extracted DeleteFileW to separate function.. I'll look for nicer solution